src/mbgl/style/expression/compound_expression.cpp

```diff
@@ -376,9 +485,9 @@ std::unordered_map<std::string, CompoundExpressionRegistry::Definition> initiali
     define("<", [](const std::string& lhs, const std::string& rhs) -> Result<bool> { return lhs < rhs; });
     define("<=", [](double lhs, double rhs) -> Result<bool> { return lhs <= rhs; });
     define("<=", [](const std::string& lhs, const std::string& rhs) -> Result<bool> { return lhs <= rhs; });
-    
+
     define("!", [](bool e) -> Result<bool> { return !e; });
-    
+
```

Comment:
Nit: revert these whitespace changes -- I think we normally use Xcode's default, which to include indentation for blank lines.

Reply:
Trailing whitespace on blank lines is not the convention. The CI bot that checked for it was turned off, but not because the convention was changed. https://github.com/mapbox/mapbox-gl-native/pull/7875

Reply:
As far as I know, the preference in mbgl is still to avoid introducing indented blank lines, while the preference in platform/{darwin,ios,macos}/ is to follow the Xcode default of indenting blank lines. It isn’t really worth fighting over invisible characters, but that goes both ways: not really worth stripping them out, and not really worth putting them back in. 🤷‍♂️

Reply:
> It isn’t really worth fighting over invisible characters, but that goes both ways: not really worth stripping them out, and not really worth putting them back in

Yeah, that's basically my stance, too: the particular preference(s) matter less than avoiding whitespace-only changes (like these ^) in diffs.